components/content-picker/index.js

```diff
@@ -186,12 +190,13 @@ ContentPicker.propTypes = {
 	contentTypes: PropTypes.array,
 	content: PropTypes.array,
 	placeholder: PropTypes.string,
-	mode: PropTypes.string,
+	mode: PropTypes.oneOf(['post', 'user', 'term']),
```

Comment:
Here and below, I think this offers more help with IntelliSense and validation in general.